helm-charts/values.yaml

```diff
@@ -248,6 +250,11 @@ wso2:
           image: wso2/adapter:latest
           security:
             sslHostname: "adapter"
+        configs:
+          tls:
+            secretName: "adapter-cert"
+            privateKey: ""
```

Comment:
are these file names of public and private keys. if so shall we make it as `privateKeyFileName` and `publicKeyFileName` to be more clear to the user? or `privateKeyDataName` and `publicKeyDataName`